tests/python/frontend/onnx/test_forward.py

```diff
@@ -85,6 +85,13 @@ def get_caffe2_output(model, x, dtype='float32'):
     return c2_out
```

Comment:
Let's just remove the caffe2 runtime and use onnxruntime everywhere instead. Although we could do it in a later PR, we might as well do it now.

Reply:
Agree and the replacement is done, thanks for suggestion.